orc8r/cloud/go/services/orchestrator/obsidian/handlers/network_handlers.go

```diff
@@ -53,6 +53,22 @@ func registerNetwork(c echo.Context) error {
 	if err != nil {
 		return echo.NewHTTPError(http.StatusBadRequest, err)
 	}
+
+	// Creating default tier for each created network
+	for _, s := range createdNetworks {
+		defaultTier := &models.Tier{
+			ID:       "default",
+			Gateways: models.TierGateways{},
+			Images:   models.TierImages{},
+			Version:  models.TierVersion("1.0"),
```

Comment:
Use this default tier version https://github.com/magma/magma/blob/0a9bf190ff22a552d99b8371b84a1abc3bb05d1c/orc8r/tools/fab/dev_utils.py#L176-L179